app/src/main/java/br/org/cesar/discordtime/stickysessions/presentation/session/SessionPresenter.java

```diff
@@ -58,4 +65,28 @@ public void onError(Throwable e) {
             }
         }, sessionId);
     }
+
+    @Override
+    public void onAddNoteClicked() {
+        if (mActiveSession != null && mActiveSession.topics != null) {
+            List<String> topics = mActiveSession.topics;
+            mView.displayAddNoteDialog(topics);
+        }
+    }
+
+    @Override
+    public void addNewNote(String topic, String description) {
+        Note note = new Note(description, "Author", topic, mActiveSession.id);
+        mAddNote.execute(new DisposableSingleObserver<Note>() {
```

Comment:
We need to show a loading while note is being created. We could call something like `mView.displayLoading()`

Reply:
One of our unit tests on this presenter could be checking that view "entered" in loading state every time we make a asynchronous call.